app/src/main/res/values/strings.xml

```diff
@@ -84,6 +85,11 @@
     <string name="links_to_social_media">Add your social media groups</string>
     <string name="my_events_screen_past_events_list_name">Past events</string>
     <string name="my_events_screen_empty_past_events_list">There is not past events yet</string>
+    <string name="join_event_failiure">Failed to Join the Event</string>
+    <string name="join_event_success">Successfully joined the event</string>
+    <string name="join_event_staff">You are already a Staff Member for this Event</string>
```

Comment:
could be shorten to Already a Staff Member for this Event.
Same for the texts afterwards